package.json

```diff
@@ -28,6 +28,7 @@
     "prerelease": "rm -rf dist/*; npm run build; zip -r dist.zip dist",
     "pretest": "npm run lint",
     "start": "vite",
+    "preview": "vite preview",
```

Comment:
`vite build && vite preview` lets you view a production version of the site locally.